Modules/Filtering/ImageIntensity/include/itkAdaptImageFilter.h

```diff
@@ -155,9 +155,7 @@ class AdaptImageFilter
   void
   SetAccessor(AccessorType & accessor)
   {
-    FunctorType functor;
-
-    functor = this->GetFunctor();
+    FunctorType functor = this->GetFunctor();
```

Comment:
For the record, here is where `AdaptImageFilter` will use the copy-constructor of `itk::Functor::AccessorFunctor` (instead of the assignment operator), which was originally deprecated, according to https://open.cdash.org/viewBuildError.php?type=1&buildid=10041267